articles/architecture-scenarios/_includes/_keep-reading.md

```diff
@@ -0,0 +1,78 @@
+<% if (platform === "b2b") { %>
```

Comment:
I would kill this and just use the platform to do the links, see below...

Reply:
yes, I have already battled this for couple of hours yesterday. The trouble is our  docs "Compiler" doesn't work correctly with interpolation and markdown links. 
It shows links if I use <%=platform%> but doesn't replace with the value. 
If I introduce space as <%= platform %>, it  does the interpolation but then dont' recognize the "line" as a markdown link. 


I didn't want to debug and fix the compiler so I duplicated the IF "platform" check and removed "platform" interpolation.